app/Models/Metric.php

```diff
@@ -221,7 +221,7 @@ public function validate()
         $messages = [];
 
         if (60 % $this->threshold !== 0) {
-            $messages[] = 'Threshold must be divisible by 60.';
+            $messages[] = 'Threshold must divide by 60.';
```

Comment:
We'd say `The threshold must divide 60.`, or we could notate this `t | 60` for `t divides 60`.